docker/cron/crontab

```diff
@@ -0,0 +1,2 @@
+0 */6 * * * sh php bin/console sylius:remove-expired-carts
```

Comment:
Shouldn't we add catalog promotion processing here as well?

Reply:
I thought I responded in the previous thread. As far as I know the catalog promotions are executed via symfony messenger. Therefore it should be run a little bit different - by a worker (bin/console messenger:consume) and it should be separated container. If I'm wrong - let me know

Reply:
you've answered but also resolved comment so I didn't see it. Seems legit 👍